app/sequencescape_excel/sequencescape_excel/column.rb

```diff
@@ -143,7 +144,7 @@ class ArgumentBuilder
       attr_reader :arguments
 
       def initialize(args, key, default_conditional_formattings)
-        @arguments = args.reverse_merge(name: key)
+        @arguments = args.merge(name: key)
```

Comment:
Wonder if this mattered? It was changed in the other direction here - https://github.com/sanger/sequencescape/commit/af0da03dd966f64463878d83c8f128aae737a591#diff-0622c2fd372210896155392045c637aa66eb184af2a6c9768e42e3f79e33df5d

Reply:
Hah, well that was the cause of the bug, or at least part of it. As we then index the columns by name, and use that for the lookup. Hence the later definitions over-writing the first.

Looks like I introduced it when attempting to ensure that donor_id2 updated donor_id, something now handled by 'updates'.